backends/pipeline_api.py

```diff
@@ -72,6 +72,7 @@ def update_profile_from_edx(backend, user, response, is_new, *args, **kwargs):
         pass
     user_profile.gender = user_profile_edx.get('gender')
     user_profile.edx_mailing_address = user_profile_edx.get('mailing_address')
+    user_profile.agreed_to_terms_of_service = True
```

Comment:
Currently the server side validation will allow you to switch this back to `False` as long as `filled_out` is not `True`. Since it doesn't seem like the user has a reason to set this anymore via PATCH, can we add it to the list of read only fields in `ProfileSerializer` and `ProfileLimitedSerializer`?